### DISCUSSION TITLE
Help: How to enable attachment cacheLimit.

### DISCUSSION THREAD

**Discussion by AosakiReiya** [powersync-ja/powersync.dart#103] [Q&A] [answered]
I don't want to synchronize all attachments when the app starts, I just want to cache some of them and add this new data to the local cache as the user accesses more historical data.

**Comment by AosakiReiya** [comment]
Is it ok if I just limit the watchIds sql query and then manually handle the sync queue myself?

**Reply by DominicGBauer** [reply, marked as answer]
Hi @SamsonMXVI yes that should work although we haven't tested this type of use case. Let us know if you run into any issues.
